packages/block-library/src/post-template/edit.js

```diff
@@ -138,7 +138,10 @@ export default function PostTemplateEdit( {
 							( { slug } ) => slug === taxonomySlug
 						);
 						if ( taxonomy?.rest_base ) {
-							accumulator[ taxonomy?.rest_base ] = terms;
+							// Skip post formats, as they are handled separately.
+							if ( ! taxonomy.rest_base === 'post_format' ) {
```

Comment:
I don't like this change as well, I think we should skip taxonomies with `show_ui` false like we do for the regular taxonomies control.

Reply:
I don't understand. This is the post preview, not the control.